values-it/strings.xml

```diff
@@ -28,47 +28,47 @@
             <string name="pref_header_interface">Interfaccia</string>
             <string name="pref_header_install">Installazione</string>
             <string name="pref_header_download">Download</string>
-            <string name="label_dark_mode">Tema scuro</string>
+            <string name="label_dark_mode">tema scuro</string>
```

Comment:
This is an option in the menu, and just as the other options, and in English, it is capitalized